File: Week2/activity/contacts.js
/*const contacts =
    [{
        name: "Jerome",
        countryCode: "+63",
        phoneNumber: "953-578-6333"
    }
    ,
    {
        name: "Ethan",
        countryCode: "+63",
        phoneNumber: "995-911-7924"
    }];


const jsonString = JSON.stringify(contacts);
console.log(jsonString);
console.log(JSON.parse(jsonString));

const phoneNumbers = contacts.map(contact => contact.phoneNumber);
console.log(phoneNumbers);*/
const fs = require('fs');

const contacts = fs.readFileSync('contacts.json','utf-8');

console.log(dataString);
console.log("-----");
console.log(JSON.parse(dataString));

File: Week2/activity/contacts.py
import json
#contacts = [
#    {
#        "name": "Jerome",
#        "countryCode": "+63",
#        "phoneNumber": "953-578-6333"
#    },
#    {
#        "name": "Ethan",
#        "countryCode": "+63",
#        "phoneNumber": "995-911-7924"
#    }
#]
#
#json_string = json.dumps(contacts)
#print(json_string)
#print(json.loads(json_string))
#phoneNumbers = []
#for contact in contacts:
#    phoneNumbers.append(contact["phoneNumber"])

#print(phoneNumbers)

file = open("contacts.json",'r')
contacts = file.read()
print(contacts)
print(json.loads(contacts))
file.close()